repo_config.json

```diff
@@ -0,0 +1,46 @@
+{
+    "ansible-collections": {
```

Comment:
this file needs to be a part of the GH environments. we can expose an example schema!

Reply:
Done. I've moved the configuration to GitHub environments.